README.md

```diff
@@ -523,55 +522,91 @@ Judgement is completed when the Judge has decided all questions and is not overr
 Judge's Judgement has been completed, any Judge is no longer a Judge and a player who can shall 
 close all open issues labelled "Judgement" in a timely fashion. 
 
-**324** *Hold it!*
+**354** *Punish the Guilty*
 
 If a player (unwittingly) performs an action that directly contradicts a
-rule, a procedure defined as a _Trial_, with the rule-breaking player as the defendant, will take place. A Trial is
-defined as follows.
-
-- Every Trial takes places on GitHub in a new issue with the "Trial" label.
-- Every Trial features a defendant.
-- Every Trial features a prosecutor.
-- Every Trial features a jury, comprised of at least one player.
-- No player can take multiple roles in a single trial, i.e. a player can not be both prosecutor and jury.
-- Both prosecution and defendant must call forth at least one player for jury duty. (Note: There is no uniqueness
-	requirement, which means a one-manned jury is still a valid jury.)
-- Every Trial shall have the name "The people vs #githubhandle XX" where #githubhandle is the name of the defendant and
-	XX is the number representing how often this player has been on Trial.
+rule, a procedure defined as a _Trial_ can take place. A Trial is concluded with a _Verdict_
+that will be enacted by someone who has this power.
+
+**359** *Roleplaying in Court*
+
+A Trial features three roles:
+
+- A defendant, the player charged with the violation.
+- A prosecutor.
+- A jury, comprised of at least one player.
+
+No player can take multiple roles in a single Trial, i.e. a player can not be both prosecutor and jury.
+
+**363** *Jury Selection*
+
+The jury in a Trial is selected in a procedure called _Jury Selection_. Jury Selection is defined as follows.
+Both prosecution and defendant must call forth at least one player for jury duty. (Note: There is no uniqueness
+requirement, which means a one-manned jury is still a valid jury.)
+
+**362** *Perfect Prosecutor*
+
+The prosecutor of a Trial is selected as follows:
+
 - If a player accuses another player of breaking a rule, the accuser will take the role of prosecutor in the Trial.
 - Else If no other method of choosing a prosecutor in a Trial is defined, the prosecutor will be selected through the
 	procedure for invoking Judgement. In this procedure the defendant will take the role of the person invoking Judgement.
```

Comment:
Ah that, well what if you turn yourself in for a violation like the good citizen you are?